docs/src/Form/index.js

```diff
@@ -28,6 +28,7 @@ class FormExample extends React.Component {
       {
         fieldType: 'password',
         name: 'Password',
+        helpBlock: 'If you want to make your password strong you could use a password manager',
```

Comment:
Changed it up a bit. Had a conversation offline and came up with something that we could both agree on